src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -36,6 +39,18 @@ public override void WriteTo(ObjectWriter writer)
                 writer.WriteBoolean(isGlobal);
             }
         }
+
+        writer.WriteInt32(_interceptsLocationInfo?.Count ?? 0);
```

Comment:
This doesn't require the version bump in AbstractSyntaxIndex.s_serializationFormatChecksum?

Reply:
it does!